### DISCUSSION TITLE
Shout-out from jetbrains blog post

### DISCUSSION THREAD

**Discussion by plusmobileapps** [badoo/Decompose#151] [Show and tell]
Just thought I'd share out the shout out from the [jetbrains kmm blog post](https://blog.jetbrains.com/kotlin/2021/05/whats-new-in-kmm-since-going-alpha/) team about Decompose. Thanks for a great library and keep up the good work 😁

> Decompose can help you share more code with KMM. Decompose provides Kotlin Multiplatform lifecycle-aware business logic components with routing functionality and a pluggable UI. Since KMM is flexible and allows you to choose how much of your codebase you want to share, you don’t have to limit its use to just your project’s core and business layers, and libraries like Decompose can help you achieve that goal.

**Comment by arkivanov** [comment]
Thanks, glad if it helps someone!
